charts/gatekeeper/templates/gatekeeper-controller-manager-deployment.yaml

```diff
@@ -49,6 +49,9 @@ spec:
         - --exempt-namespace={{ .Release.Namespace }}
         - --operation=webhook
         - --enable-mutation={{ .Values.experimentalEnableMutation}}
+        {{- range .Values.disabledBuiltins}}
```

Comment:
This file is copied from manifest staging when we cut a new release. We can revert this change.